index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Bowling Scorecard MVP</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <style>
+
+    </style>
+  </head>
+  <body>
+    
+    <h1> B O W L I N G </h1>
+    
+    <br><br>
+    
+
+    <a href="#" label="addRoll">
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      Add roll
+    </a>
+
```

Comment:
Simplistic yet visually pleasing web app so far - great that you managed to include some styling given the time constraints of this challenge!